### PR TITLE
Increase activation to 80%

### PR DESCRIPTION
**Pull request by jgarzik**

### CODE REVIEW

**Comment by jtoomim** [comment]
bigblocks.py would need to be updated if the activation threshold were changed.

That said, I do not support changing the activation threshold. If some miners want to only vote for this if a larger supermajority also supports it, I think the best way to achieve that goal is to mine blocks with version 0x20000000 instead of 0x30000000. When each miner sees that the 0x20.. + 0x30... blocks reach whatever arbitrary threshold they decide is "enough", then they can talk to each other and get all of the 0x20... miners to switch at the same time to 0x30.

**Comment by ojanssens** [comment]
This needs to be debated first. Closed for now.

**Comment by Leviathn** [comment]
at 75%, threshold remains _substantially_ lower than the threshold used for the last soft fork (95%).

**Comment by benjiqq** [comment]
I'd be very curious to know how one can _enforce_ such a rule per algorithm (longest chain rule)? Basically Bitcoin is at a stage where developer teams cater to mining cartels explicitly.

**Comment by dskloet** [comment]
Jonathan, are you still talking to the big miners? Do you know who is willing to support at which threshold? Do they understand about using the different version bit as you suggest?

If 80% is willing to support a threshold of 80% but only 50% is willing to support a threshold of 75%, then 80% is clearly a winner. I think the main strength of Classic is that it started from a dialog with the miners and I hope that continues to be its strength.

File: src/chainparams.cpp [hunk 1]
```diff
@@ -56,7 +56,7 @@ class CMainParams : public CChainParams {
 
         // Timestamps for forking consensus rule changes:
         // Allow bigger blocks if:
-        consensus.nActivateSizeForkMajority = 750; // 75% of hashpower to activate fork
+        consensus.nActivateSizeForkMajority = 800; // 80% of hashpower to activate fork
```
Comment (dskloet) [inline src/chainparams.cpp:59]: The comment is actually not accurate. Even with just 77% hashpower there's a 60% chance of activating within 90 days.
I think it's clear enough if the comment says  // of 1000 most recent blocks

**Comment by seweso** [comment]
Consider.it page: https://bitcoinclassic.consider.it/increase-activation-to-80

**Pull request closed without merging**